### ISSUE TITLE
Full screenshot as blank at bottom

### ISSUE THREAD

**Issue by lannn** [route/ferrum#31]
First, thank you for writing this gem. I wrote some samples and got some good results. 
But when I use the option full as true. It causes the bottom as blank below:

Run:

```
require "ferrum"

browser = Ferrum::Browser.new
browser.goto("https://google.com")
search_input = browser.at_css("input[name='q']")
search_input.focus.type("Gym", :Enter)

browser.screenshot(path: "google.jpg", full: true)
```

Result:
![google](https://user-images.githubusercontent.com/4487532/67666708-edb93e00-f99e-11e9-8cb6-bd950ad1269d.png)

**Comment by route** [comment]
It has been fixed in master, use repo from guthub for now.

**Comment by route** [comment]
Fixed by #29

**Comment by route** [comment]
Released new version, it should be ok using gem!
